_projects/hookshot

```diff
@@ -0,0 +1,60 @@
+---
+name: Hookshot Bridge
+desc: Extend our webhook bridge to support more things!
+# add a short one line description of your project
+requirements:
+ - Experience with JavsScript / Typescript is helpful
+ - Some familiarity with writing chat bots / services.
+difficulty: medium
+issues:
+ - https://github.com/matrix-org/synapse/issues/4556
+mentors:
+# First person in contact; mentors may change before project starts.
+ - Half-Shot
+initiatives:
+ - GSoC
+tags:
```

Comment:
You need a short or long tag. this one is long:
```suggestion
tags:
   - 350h
```

Reply:
Oh sure. I would probably put it around the 200h mark though.

Reply:
Oh they come in two flavours. 350 or 180 iirc

Reply:
:o